string target_robot
string message